tests/testthat/test-custom-remote.R

```diff
@@ -0,0 +1,12 @@
+context("remote")
+
+setwd("./travis-testthat")
+
+test_that("custom remote is honored", {
+
+  system("git remote add test https://github.com/pat-s/travis-testthat.git")
+
+  expect_is(github_info(remote = "test"), "gh_response")
```

Comment:
`expect_s3_class()` is now the recommended test.

Reply:
Done.